### DISCUSSION TITLE
How to remove "everyting" plugin

### DISCUSSION THREAD

**Discussion by NolanNolan** [Flow-Launcher/Flow.Launcher#1413] [Q&A] [answered]
Hi

title says it all, there is no "delete" button next to the plugin in flow launcher setting, so how to i remove it again ?

How is everything plugin to be used, i see no difference when i type some search string into flow launchers input field ?

thanks

Nolan

**Comment by taooceros** [comment, marked as answer]
Hi you can type pm uninstall `pluginname` to uninstall. 

Everything plugin use everything as a backend to perform file or folder search. Sometimes it works better than the Windows index search used by explorer plugin.

**Comment by NolanNolan** [comment]
ok thanks, that worked. 

So i wont notice that everything is used instead of the build in search ? Theres no way specific to invoke or use everything pkugin ?

**Reply by taooceros** [reply]
You will? Sometimes the Windows index won't find all files, but everything will. So if you use the everything plugin you may be able to search more file. Also you can check which plugin is responsible for a specific Result by shift enter.

**Reply by jjw24** [reply]
Just to clarify though, by its default settings Windows Index may not find all your files, but you can configure it to find in additional directories by configuring the indexing options: https://github.com/Flow-Launcher/Flow.Launcher/discussions/1413#discussioncomment-3739385

**Reply by deefrawley** [reply]
> Theres no way specific to invoke or use everything pkugin ?

Give it a keyword, then you can choose when you want to explicitly use it to search (that's what I do).

**Reply by NolanNolan** [reply]
ok thanks

**Comment by NolanNolan** [comment]
ok thanks 👍

But it seems the plugin cannot index content in eg documents, or am i missing something ?

Nolan

**Reply by onesounds** [reply]
![image](https://user-images.githubusercontent.com/6903107/192421577-c149f44b-bab5-403a-87ac-f566b1d3cbd2.png)
- Check the this area. (when you done. press F5 in querybox.)

**Reply by NolanNolan** [reply]
thanks but thats "only" the normal windows search right ?

I meant its not opossible to make "everythong" plugin index file content if so how ?

**Reply by jjw24** [reply]
Yes, I think it is the case the Everything service that the plugin uses doesn't index file content, but I am not sure whether the latest Everything service release is still the case. I think @taooceros you may have mentioned the latest Everything service release does do file contents but sdk has not been updated to include this functionality?

**Comment by NolanNolan** [comment]
I see thanks 😊


Hilsen Jørn

Fra: Kevin ***@***.***>
Sendt: 26. september 2022 01:06
Til: ***@***.***>
Cc: Jørn Bue ***@***.***>; ***@***.***>
Emne: Re: [Flow-Launcher/Flow.Launcher] How to remove "everyting" plugin (Discussion #1413)


You will? Sometimes the Windows index won't find all files, but everything will. So if you use the everything plugin you may be able to search more file. Also you can check which plugin is responsible for a specific Result by shift enter.
